shared/clinical-code-sets/tests/haemoglobin/1/README.md

```diff
@@ -0,0 +1,17 @@
+# Haemoglobin
+
+A patient's haemoglobin as recorded via clinical code and value. This code set only includes codes that are accompanied by a value (`423.. - Haemoglobin estimation`). It does not include codes that indicate a patient's haemoglobin (`4235 - Haemoglobin low`) without giving the actual value.
+
+Haemoglobin codes retrieved from https://www.medrxiv.org/content/medrxiv/suppl/2020/05/19/2020.05.14.20101626.DC1/2020.05.14.20101626-1.pdf and metadata available in this directory.
```

Comment:
Similar to my comment in the ferritin pull request... When we talk about metadata for a code set, we don't mean the list of codes - instead we mean the description of how the set was constructed. Typically this is json files downloaded from GetSet. So just remove the bit about metadata from this line.